aip/auth/4110.md

```diff
@@ -165,6 +176,54 @@ digraph d_front_back {
     1. If a quota project is provided explicitly while initiating ADC, override the quota project in the credential with the explicit value. _[END]_
     1. Else if the `GOOGLE_CLOUD_QUOTA_PROJECT` environment variable is set, override the quota project in the credential with this value. _[END]_
 
+### Retrieval and Caching of Workload Access Tokens from the Metadata Server
```

Comment:
This section looks great! But this is not the best place to put it. There are separate AIPs per credential/backend type. This content can go in https://google.aip.dev/auth/4115

Reply:
OK, moved to 4115.